src/lib/graphqlTimeoutMiddleware.ts

```diff
@@ -62,25 +72,32 @@ export const graphqlTimeoutMiddleware = (defaultTimeoutInMS: number) => {
     }
     // TODO: Maybe cache if it turns out to take significant time.
     //       Should probably be cached on the schema instance.
-    const timeoutInMS = timeoutForField(fieldFromResolveInfo(info)) || defaultTimeoutInMS
+    const timeoutInMS =
+      timeoutForField(fieldFromResolveInfo(info)) || defaultTimeoutInMS
     let timeoutID
     return Promise.race([
       new Promise((_resolve, reject) => {
         timeoutID = setTimeout(() => {
-          const field = `${info.parentType}.${info.fieldName}`;
-          reject(new GraphQLTimeoutError(`GraphQL Timeout Error: ${field} has timed out after waiting for ${timeoutInMS}ms`))
+          const field = `${info.parentType}.${info.fieldName}`
+          reject(
+            new GraphQLTimeoutError(
+              `GraphQL Timeout Error: ${
+                field
+              } has timed out after waiting for ${timeoutInMS}ms`
+            )
+          )
         }, timeoutInMS)
       }),
       resolverResult.then(
         result => {
           clearTimeout(timeoutID)
-          result
+          return result
```

Comment:
This is the fix.

Reply:
Ah, funny that this didn't get flagged from @mbilokonsky linting work. 

As to formatting, your prettier is working now!

Reply:
👍

Reply:
Nice one

Reply:
Oh man, I don’t get how there wasn’t any tests failing for this 🤔 Did you take a look at why they have been insufficient?

Reply:
likely that the queries running in unit tests don't have the middleware applied

Reply:
I'm gonna try write a single integration test tomorrow on metaphysics to catch this kind of thing